src/components/NewArticle/index.js

```diff
@@ -0,0 +1,311 @@
+import React from 'react';
+import Downshift from 'downshift';
+import PropTypes from 'prop-types';
+
+import ReactQuill from 'react-quill';
+
+import { WithContext as ReactTags } from 'react-tag-input';
+import { sampleCategoryOptions, sampleTagOptions } from '../../../mockdata/samplebody';
+
+const KeyCodes = {
+  comma: 188,
+  enter: 13,
+  semicolon: 186
+};
+const delimiters = [KeyCodes.comma, KeyCodes.enter, KeyCodes.semicolon];
```

Comment:
Would we not achieve the same with `Object.values(KeyCodes)`

Reply:
Yes, changed this.